c/pdsutil.c

```diff
@@ -150,7 +149,7 @@ void listDirectory(char *pdsName){
 
   sprintf(filenameBuffer,"//'%s'",pdsName);
   in = fopen(filenameBuffer,"rb");
-  zowelog(NULL, LOG_COMP_RESTDATASET, ZOWE_LOG_DEBUG, "fopen in=0x%x errno=%d\n",in,errno);
+  printf("fopen in=0x%x errno=%d\n",in,errno);
```

Comment:
Your previous comment stated that we should remove zowelogs from pdsutils

Reply:
Change these to debug

Reply:
We need to have some way to return diagnostics info, so it's either rc/rsn or messages. Since we don't have rc/rsn as `mtlskt.c` does, we have to have some messages (we can change the API). Naked `printf`s are bad in library code, so the best we can do is keep `zowelog`. 

In pdsutil.c we should:
* Keep `zowelog` and use `ZOWE_LOG_DEBUG` so they aren't printed without tracing on
* Change `printf` and `dumbuffer` to `zowelog` and `zowedump` with `ZOWE_LOG_DEBUG`